packages/ui/cypress/e2e/app/agents/read/form.js

```diff
@@ -0,0 +1,29 @@
+export default () =>
+  describe('form content', () => {
+    beforeEach(() => {
+      cy.goToRoute('/app/agents?filterColumn=&itemId=2&mainColumn=edit')
```

Comment:
change to visit, since we will not use login() (which does a visit internally) (goToRoute uses react router to pushState and it is not possible before a first visit has been done)